### PR TITLE
[Travis] Added Behat job

### PR DESCRIPTION
**Pull request by mnocon**
Things done:
1) Added a simple Behat job running the tests covering System Information part.
2) Split the current setup into two separate jobs: CS and unit tests
3) Added Slack notifications

~~Requires https://github.com/ezsystems/ezplatform-admin-ui/pull/1647 to be merged first.~~

### CODE REVIEW

**Comment by mnocon** [comment]
Looks like I've made too many formatting changes, I will revert them to make this PR easier to review

**Review by tomaszszopinski — APPROVED** [review]
lgtm

**Review by micszo — APPROVED** [review]
🚀

File: .travis.yml [hunk 1]
```diff
@@ -11,25 +9,55 @@ matrix:
     fast_finish: true
     include:
         - php: 7.3
+          name: "Code style"
           env: CHECK_CS=true
+        - php: 7.3
+          name: "Unit tests"
+          env: UNIT_TESTS=true 
+        - php: 7.3
+          name: "System informarion browser tests"
```
Comment (konradoboza) [inline .travis.yml:18]:
```suggestion
          name: "System information browser tests"
```

**Review by konradoboza — APPROVED** [review]

**Pull request merged**